.circleci/config.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+jobs:
+    build:
+        branches:  
+            ignore:   
+                - master
+        machine: true
+        steps:
+            - checkout
+
+            # build the application image
+            - run: |
+                  echo "$DPASS" | docker login --username $DUNAME --password-stdin
+                  cd litmus-portal && make build push
```

Comment:
Remove This part

Reply:
It shouldn't be a problem as the the workflow will not be executed in the master, also we need some run command or ci throws an error. I have added a simple echo as a run command.